src/components/Tag/Tag.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import _ from 'lodash';
+import { lucidClassNames } from '../../util/style-helpers';
+import CrossIcon from '../Icon/CrossIcon/CrossIcon';
+import {
+	createClass,
+	filterTypes,
+	rejectTypes,
+	omitProps,
+} from '../../util/component-types';
+
+const cx = lucidClassNames.bind('&-Tag');
+
+const {
+	bool,
+	func,
+	node,
+	string,
+	object,
+} = React.PropTypes;
+
+/**
+ * {"categories": ["communication"]}
+ *
+ * Organizes items into removable items or groups. Can be nested to facilitate
+ * grouping.
+ */
+const Tag = createClass({
+	displayName: 'Tag',
+
+	propTypes: {
+		/**
+		 * Shows or hides the little "x" for a given tag.
+		 */
+		showXButton: bool,
```

Comment:
Maybe we should rename this to `isRemovable` to match Selection. There's also `isCloseable` in lucid but that's not very good IMO.